src/Package/Impl/DataInspect/DataSource/GridDataSource.cs

```diff
@@ -26,28 +27,13 @@ public class GridDataSource {
 
             REvaluationResult? result = null;
             using (var evaluator = await rSession.BeginEvaluationAsync()) {
-                result = await evaluator.EvaluateAsync($"rtvs:::grid.dput(rtvs:::grid.data({expression}, {rows}, {columns}))", REvaluationKind.Normal);
+                result = await evaluator.EvaluateAsync($"rtvs:::toJSON(rtvs:::grid.data({expression}, {rows}, {columns}))", REvaluationKind.Normal);
```

Comment:
You don't need to call `toJSON` anymore. Instead, pass `REvaluationKind.Json` as the evaluation flag, and then use `REvaluationResult.JsonResult`.